baseapp/abci.go

```diff
@@ -312,6 +314,37 @@ func (app *BaseApp) deliverTxWithoutEventHistory(req abci.RequestDeliverTx) (res
 	}
 }
 
+// checkHalt checkes if height or time exceeds halt-height or halt-time respectively.
+func (app *BaseApp) checkHalt(height int64, time time.Time) {
+	var halt bool
+	switch {
+	case app.haltHeight > 0 && uint64(height) > app.haltHeight:
+		halt = true
+
+	case app.haltTime > 0 && time.Unix() > int64(app.haltTime):
+		halt = true
+	}
```

Comment:
`switch` seems too verbose here.
```suggestion
	halt := (app.haltHeight > 0 && uint64(height) > app.haltHeight) ||
		(app.haltTime > 0 && time.Unix() > int64(app.haltTime))
```
or
```suggestion
	var halt bool
	if app.haltHeight > 0 && uint64(height) > app.haltHeight {
		halt = true
	} else if app.haltTime > 0 && time.Unix() > int64(app.haltTime) {
		halt = true
	}
```

Reply:
Thanks!